Dockerfile

```diff
@@ -3,8 +3,8 @@ FROM madnificent/ember:3.28.5 as builder
 LABEL maintainer="info@redpencil.io"
 
 WORKDIR /app
-COPY package.json .
-RUN npm install
+COPY package-lock.json .
```

Comment:
The package.json still needs to be copied as well I think, but not 100% sure.

I did it like that in Loket at least:
https://github.com/lblod/frontend-loket/commit/b47eb8951ab4702d5eb4ec75ce026094016845d0#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R6